prow/apis/prowjobs/register.go

```diff
@@ -19,4 +19,12 @@ package prowjobs
 const (
 	// GroupName defines the API group identifier for prow types.
 	GroupName = "prow.k8s.io"
+
+	// Kind defines the API kind identifier.
+	Kind = "ProwJob"
+)
+
+var (
+	// KindPlural is the plural form of Kind
```

Comment:
if you ask me, I would just create constant with "ProwJobs" eg Kinds = "ProwJobs"

Reply:
This follows the kind / kindPlural pattern of CRDs and would allow users to choose the correct constant for selectors.

https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/